executor/show_test.go

```diff
@@ -1119,6 +1119,30 @@ func (s *testSuite5) TestShowClusterConfig(c *C) {
 	c.Assert(tk.QueryToErr("show config"), ErrorMatches, confErr.Error())
 }
 
+func (s *testSuite5) TestShowConfigHideIndexUsage(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+
+	var confItems [][]types.Datum
+	var confErr error
+	var confFunc executor.TestShowClusterConfigFunc = func() ([][]types.Datum, error) {
+		return confItems, confErr
+	}
+	tk.Se.SetValue(executor.TestShowClusterConfigKey, confFunc)
+	strs2Items := func(strs ...string) []types.Datum {
+		items := make([]types.Datum, 0, len(strs))
+		for _, s := range strs {
+			items = append(items, types.NewStringDatum(s))
+		}
+		return items
+	}
+	confItems = append(confItems, strs2Items("tidb", "127.0.0.1:1111", "log.level", "info"))
+	confItems = append(confItems, strs2Items("tidb", "127.0.0.1:1111", "index-usage-sync-lease", "0"))
```

Comment:
Please add one more test case whose config name is upper-case like `INDEX-USAGE-SYNC-LEASE`.